linux/setup_repo.sh

```diff
@@ -7,6 +7,57 @@ if [[ ${EUID} -ne 0 ]]; then
      exit 1
 fi
 
+if [ -x "$(command -v lsb_release)" ]; then
+    ISUBUNTU=`lsb_release -is`
+else
+    echo "lsb_release not found. Cannot determine Linux distribution (or this is not Linux)."
+    exit 1
+fi;
+
+apt --version > /dev/null
+if [[ $? -eq 0 ]]; then
+    APT_VERSION=`apt --version`
+    APT_MAJOR=$(echo $APT_VERSION| cut -d' ' -f 2 | cut -d'.' -f 1)
+    APT_MINOR=$(echo $APT_VERSION| cut -d' ' -f 2 | cut -d'.' -f 2)
+    echo "Apt version: ${APT_VERSION}"
+else
+    echo "Apt is not available."
+    exit 1
+fi;
+
+if (( $(echo "${APT_MAJOR}.${APT_MINOR} < 2.4" | bc -l) )); then
+    echo "Apt version incompatible. Cannot install repository, but you can use the .deb package to manually install."
+    exit 1
+fi;
+
+if [[ $ISUBUNTU == "Ubuntu" ]]; then
+
+    UBUNTU_VERSION=`lsb_release -sr`
+    echo "Ubuntu version: ${UBUNTU_VERSION}"
+    SUBSTRING=$(echo $UBUNTU_VERSION| cut -d'.' -f 1)
+
+if [[ $SUBSTRING -lt 22 ]]; then
+cat << EOM
+WARNING: Ubuntu versions less that 22.04 have a bug in their apt version for signed repositories.
+Not adding jamulus repo. Either install the deb file manually [see https://jamulus.io/wiki/Installation-for-Linux]
+or update your system to at least Ubuntu 22.04
+EOM
+    exit 1
+fi;
+
+else
+    echo "This is not Ubuntu, it is ${ISUBUNTU}"
```

Comment:
I think you can get rid of that and just check for the apt version

Reply:
Good point, keep it simple.
Updated.